overlord/ifacestate/ifacemgr.go

```diff
@@ -277,6 +277,24 @@ func getPlugAndSlotRefs(task *state.Task) (*interfaces.PlugRef, *interfaces.Slot
 	return &plugRef, &slotRef, nil
 }
 
+func getConns(task *state.Task) (map[string]connState, error) {
+	// Get information about connections from the state
+	var conns map[string]connState
+	err := task.State().Get("conns", &conns)
```

Comment:
The reason it takes a task is that task can log things in a way that will be hopefully useful. Should I just not log anything at all or use a different logging mechanism?